apps/charterafrica/src/payload/globals/Ecosystem.js

```diff
@@ -90,6 +90,13 @@ const Ecosystem = {
                     },
                   },
                 }),
+                airtableColumnSelect({
```

Comment:
avatarUrl seem to repeat in a few places, can't we make a standalone field for it?